mathesar/tests/views/api/test_record_api.py

```diff
@@ -42,6 +51,40 @@ def test_record_list(create_table, client):
         assert column_name in record_data
 
 
+def test_record_list_filter(create_table, client):
```

Comment:
I think it would be good to document somewhere how to use the filters from an API perspective – format, operations supported, etc.

Reply:
Do we have somewhere we've put similar things? It is literally the `sqlalchemy-filters` format so I think linking to [those docs](https://github.com/centerofci/sqlalchemy-filters) might be enough, but I'm not sure where to put it.

Reply:
Somewhere in the comments is fine for now, it's on my todo list to set up proper API docs: https://github.com/centerofci/mathesar/issues/69